docs/partner_editable/deployment_options.adoc

```diff
@@ -53,6 +53,86 @@ IMPORTANT: Choose the first option if you can create an IAM role in your account
             ],
             "Resource": "*"
         }
----- 
-
-
+----
```

Comment:
Doc changes needs to be made as a separate PR to doc-edits branch. We are using separate branches for docs and code. 
Doc changes goes to doc-edits, and code changes goes to main.

Reply:
Thanks Shivansh. I've reverted the doc changes for now and I'll push a new PR in a few days. Could you please look into merging the code changes ASAP?